lib/jekyll/plugin_manager.rb

```diff
@@ -82,6 +82,14 @@ def plugins_path
       if site.config["plugins_dir"].eql? Jekyll::Configuration::DEFAULTS["plugins_dir"]
         [site.in_source_dir(site.config["plugins_dir"])]
       else
+        custom_plugins_path
+      end
+    end
+
+    def custom_plugins_path
+      if Jekyll::Utils::Platforms.windows?
+        Array(site.config["plugins_dir"])
```

Comment:
@parkr to an extent it is, except Pathname is entirely broken on Windows when it comes to paths so it starts complaining and breaking Jekyll.